xml/decoders/Public_Domain_dccdoma_UNI16ARD_8VYH.xml

```diff
@@ -24,7 +25,11 @@
     <decoder>
 
         <family name="dccdoma.cz" mfg="Public-domain and DIY" >
-            <model model="UNI16ARD-8VYH" lowVersionID="40" highVersionID="59" productID="16843009" maxInputVolts="24V AC" maxTotalCurrent="1.0A peak 3.3A" numOuts="16" connector="other" />
+            <model model="UNI16ARD-8VYH"   lowVersionID="00" highVersionID="39" productID="16843009" maxInputVolts="24V AC" maxTotalCurrent="1.0A peak 3.3A" numOuts="16" connector="other" />
+            <model model="UNI16ARD-8VYH-4" lowVersionID="40" highVersionID="49" productID="16843009" maxInputVolts="24V AC" maxTotalCurrent="1.0A peak 3.3A" numOuts="16" connector="other" />
+            <model model="UNI16ARD-8VYH-5" lowVersionID="50" highVersionID="59" productID="16843009" maxInputVolts="24V AC" maxTotalCurrent="1.0A peak 3.3A" numOuts="16" connector="other" />
+            <model model="UNI16ARD-8VYH-6" lowVersionID="60" highVersionID="69" productID="16843009" maxInputVolts="24V AC" maxTotalCurrent="1.0A peak 3.3A" numOuts="16" connector="other" />
+            <model model="UNI16ARD-8VYH-7" lowVersionID="70" highVersionID="79" productID="16843009" maxInputVolts="24V AC" maxTotalCurrent="1.0A peak 3.3A" numOuts="16" connector="other" />
         </family>
```

Comment:
The old line had lowVersionID = 40 and highVersionID = 59. None of the new lines have this combination.
Is this correct? Just checking.

Reply:
Yes,it is correct. 
In old version was decoders identified by <qualifier> Version. In condition all items must be true. It's a discontinuous interval problem.
In current version are decoders identified by include="model1,model2". In condition at last one model must be found.
Qualifier was replaced by include.